Assets/MixedRealityToolkit-SDK/Inspectors/UX/Interactable/ThemeInspector.cs

```diff
@@ -257,11 +257,14 @@ public static SerializedProperty ChangeThemeProperty(int index, SerializedProper
                 {
                     renderHost = (GameObject)target.objectReferenceValue;
                 }
-
+                
```

Comment:
```suggestion

```

Reply:
Not sure if there's a suggestion here. I deleted a tab from this area.

Reply:
Looks like some spaces were added